### DISCUSSION TITLE
Lighting Training No Progress

### DISCUSSION THREAD

**Discussion by Chin-Shan-Hong** [Lightning-AI/pytorch-lightning#19914] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
I want to train a custom document classification model with LayoutLMv3. And I am following the tutorial here https://www.mlexpert.io/blog/document-classification-with-layoutlmv3

But I am not sure why my training has no progress for more than 30 minutes, is this normal?

![image](https://github.com/Lightning-AI/pytorch-lightning/assets/163954492/c9103f23-04fb-4aa6-9343-2a02146ee4e0)

**Comment by snknitin** [comment]
Did you find a fix? In my case RichProgressBar is not visible for some reason
